blueprints/disruption-budgets/README.md

```diff
@@ -118,13 +134,16 @@ Let's say you want to control how nodes are upgraded when switching to Bottleroc
 To deploy the Karpenter NodePool and the sample workload, simply run this command:
 
 ```
+sed -i '' "s/<<CLUSTER_NAME>>/$CLUSTER_NAME/g" nodepool-restrictive-budget.yaml
+sed -i '' "s/<<KARPENTER_NODE_IAM_ROLE_NAME>>/$KARPENTER_NODE_IAM_ROLE_NAME/g" nodepool-restrictive-budget.yaml
 kubectl apply -f .
 ```
 
 You should see the following output:
 
 ```
-nodepool.karpenter.sh/default created
+nodepool.karpenter.sh/restrictive-budget created
+ec2nodeclass.karpenter.k8s.aws/restrictive-budget created
 deployment.apps/workload-multi-az-nodes created
```

Comment:
This blueprint is still not working because of this. Pods are deployed to the nodes launched by the default NodePool, so any change you make in this new NodePool, it's not reflected within the sample workload.